src/WebUI/Components/ServicesComponent.tsx

```diff
@@ -202,4 +204,25 @@ export class ServicesComponent extends BaseComponent<IServicesComponentPropertie
 
         return ListComponent.renderColumn(textToRender || "", ListComponent.defaultColumnRenderer, colDataClassName);
     }
+
+    private static _getServiceStatus(service: IServiceItem, cssClassName: string): React.ReactNode {
+        let statusProps: IStatusProps = Statuses.Success;
+        let tooltipText: string | undefined;
+        if (service.type == "LoadBalancer") {
+            if (service.externalIP == "") {
+                tooltipText = Resources.ExternalIPAllocPending;
+                statusProps = Statuses.Running;
+            }
+            tooltipText = Resources.ExternalIPAllocated;
+        }
+
+        return (
+            <div>
+                <PodStatusComponent statusProps={statusProps}

```

Comment:
keep this in 222 line
statusProps={statusProps}

Reply:
Done.